### DISCUSSION TITLE
[Enhancement] Cast support

### DISCUSSION THREAD

**Discussion by hyd3rs** [ciderapp/Cider#664] [Added Features]
**Is your feature request related to a problem? Please describe.**
I usually listen to music with two Google Home speakers in my office. While the Apple Music app on Android works just fine for this, I would like to have a way to control the music playing from my computer.

**Describe the solution you'd like**
A Chromecast/Google Cast feature to stream music from Cider to a Cast-enabled device (or just Cast-enabled speakers)

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by quacksire** [comment]
We have support for casting to Google Home, we literally had to buy one to test it (_cough_ @cryptofyre _cough_)

What we were working in the AME era was Chromecast support.

Soon.

**Comment by hyd3rs** [comment]
Oh? I don't see any Cast button - does the feature in its' current state not support speaker pairs? Even then, I'd still assume that there would be a Cast button since I have other standalone Nest/Google Home devices.

**Comment by tva2000hn** [comment]
It’s not committed yet

**Comment by Amaru8** [comment]
Chromecast was implemented a long time ago, locking discussion.
